app/boot.py

```diff
@@ -117,8 +117,6 @@ def add_default_presets():
                                                               {'name': 'mesh-size', 'value': '300000'}]})
         Preset.objects.update_or_create(name='Buildings', system=True,
                                         defaults={'options': [{'name': 'mesh-size', 'value': '300000'},
-                                                              {'name': 'use-3dmesh', 'value': True},
-                                                              {'name': 'pc-geometric', 'value': True},
```

Comment:
You can leave pc-geometric for buildings (it's probably a good one to have).

Reply:
Good! I was testing that locally and it seems to help reconstruction fidelity a bit. Things are a bit less wibbly-wobbly with it on.